test/README.md

```diff
@@ -0,0 +1,10 @@
+# Forseti Infrastructure Tests
```

Comment:
This documentation is a bit sparse and may be easy to miss. Can this be pushed into the README?